test/packages/knowledge_base/kibana/kb_entry/foo/index-mapping.json

```diff
@@ -0,0 +1,42 @@
+{
+  "dynamic": "strict",
```

Comment:
Is this setting going to be the same in all knowledge bases?

Reply:
Not *necessarily*, I mean given the mappings are defined per KB entry, I assume we would be fine letting this be the source of truth for the `dynamic`allocation too (given it's technically a part of the mapping, not a setting).

But having this "hardcoded" and non-configurable by the package would be fine too ihmo

Reply:
> But having this "hardcoded" and non-configurable by the package would be fine too ihmo, if we want to switch to the `fields.yml` approach

Yes, if we use the `fields.yml` approach this could be a setting managed by Fleet, hardcoded by now to `strict`. If needed for some future case we could add a setting to the knowledge base manifest to control the value.